frontend/app/components/ui/Toast/hooks/useToastBuilder.ts

```diff
@@ -0,0 +1,32 @@
+import { useState } from "react";
+import { ToastContent, ToastData, ToastPublisher } from "../domain/types";
+import { generateToastUniqueId } from "../helpers/generateToastUniqueId";
+
+// ⚠️ You should not probably import this. use `useToast` instead
+export const useToastBuilder = () => {
+  const [toasts, setToasts] = useState<ToastContent[]>([]);
+
```

Comment:
Can you update `toggleToast` function to use the id of the toasts instead of index?

Reply:
oops wrong code is showing 😅